source/django_package/domain/import_datas/intermediate_layer.py

```diff
@@ -0,0 +1,18 @@
+from source.django_package.domain.import_datas import parsing_config_file
+from source.django_package.domain.import_datas.import_csv import import_csv_data
+
+
+class Adapter:
+    data_type = parsing_config_file.ReadConfigFile.data_type[0]
+
+    def get_data_type(self):
+        if self.data_type == 'csv':
+            import_csv_data.get_data()
+        elif self.data_type == 'json':
+            pass
+        elif self.data_type == 'xls':
+            pass
```

Comment:
In general people avoid such if/elif/elif/... statements. It's ugly, it mixes the business logic with the code and it's hard to maintain. How to avoid it? Option 1:
`parse_config_file` should return an object that "knows" which `import_*_data` to use; then you simply do:
```python
data_importer = self.config_parser.get_data_importer()
data_importer.get_data()
```
Option 2
You define a mapping with types as keys and importers as values (and this can go into a class property):
```python
class Adapter(object):
    _importers = {
        'csv': import_csv_data
    }
    def get_data_type(self):
        importer = self._importers[self.data_type]
        importer.get_data()
```